google/cloud/bigquery/v2/minimal/benchmarks/benchmarks_config.cc

```diff
@@ -17,6 +17,7 @@
 #include "google/cloud/internal/build_info.h"
 #include "google/cloud/internal/compiler_info.h"
 #include "google/cloud/internal/getenv.h"
+#include "google/cloud/internal/make_status.h"
```

Comment:
Unused, and there is a naked `Status()` constructor below.

Reply:
Don't know what happened. Fixed.